CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Added
+
+- Web Content Accessibility (WCAG) 2.0
+- Slider controls for Web Content Accessibility
```

Comment:
Isn't the latter included in the first one?

Reply:
Yes but it was added as a separate issue. And would benefit users to know that there are slider controls now implemented (people who don't need WCAG

Reply:
Issues are internal and unrelated to the public changelog. "Slider controls" is a super generic term and could mean anything - this doesn't help anybody in understanding what exactly has been added. E.g. "keyboard controls" would be much more helpful.